### PR TITLE
Update bcftools stats: tectonic dependency

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

This PR updates the tectonic dependency, required for fixing a bug related to the report generation.

### CODE REVIEW

File: tools/bcftools/bcftools_stats.xml [hunk 1]
```diff
@@ -89,13 +89,13 @@ ${section.verbose}
...
-            <param name="first_allele_only" type="boolean" truevalue="--1st-allele-only" falsevalue="" label="1St Allele Only" 
-                   help="Include only 1st allele at multiallelic sites" />
+        <section name="sec_default" expanded="true" title="Stats options">
+            <param name="first_allele_only" type="boolean" truevalue="--1st-allele-only" falsevalue="" label="First allele only" 
+                   help="Include only first allele at multiallelic sites" />
             <conditional name="depth">
-                <param name="set_depth" type="select" label="depth distribution: min,max,bin_size [0,500,1]">
-                    <option value="no">use depth defaults</option>
-                    <option value="yes">set depth</option>
+                <param name="set_depth" type="select" label="Depth distribution" help="The default values are: min=0, ,max=500, bin_size=1">
```
Comment (nsoranzo) [inline tools/bcftools/bcftools_stats.xml:96]:
```suggestion
                <param name="set_depth" type="select" label="Depth distribution">
```

**Review by nsoranzo — APPROVED** [review]

**Comment by nsoranzo** [comment]
Another tool fails linting:
```
2023-01-11T15:32:04.6378420Z +Linting tool /home/runner/work/tools-iuc/tools-iuc/tools/bcftools/bcftools_view.xml
2023-01-11T15:32:04.8002397Z Failed linting
2023-01-11T15:32:04.8154678Z Applying linter tests... FAIL
2023-01-11T15:32:04.8155117Z .. ERROR: Test 4: Test param regions not found in the inputs
```

File: tools/bcftools/bcftools_view.xml [hunk 2]
```diff
@@ -268,7 +268,13 @@ ${section.drop_genotypes}
...
             <param name="apply_filters" value="PASS" />
             <param name="samples" value="NA00003" />
-            <param name="regions" value="20,Y" />
+            <conditional name="regions">
+                <param name="regions_src" value="regions"/>
+                <repeat name="regions">
+                    <param name="chrom" value="Y" />
+                    <param name="start" value="20" />
+                </repeat>
+            </conditional>
```
Comment (nsoranzo) [inline tools/bcftools/bcftools_view.xml:277]: Should this also be inside a `<section name="sec_restrict">` (also in at least another test)?

Comment (gallardoalba) [reply tools/bcftools/bcftools_view.xml:277]: Yes, you are completely right.

**Review by nsoranzo — APPROVED** [review]
You may want to use "Squash and merge" or "Rebase and merge" when merging.

**Comment by gallardoalba** [comment]
> You may want to use "Squash and merge" or "Rebase and merge" when merging.

Yes, I think it is a good idea. Do you think this PR can be merged?

**Pull request merged**